backend/Dockerfile

```diff
@@ -28,16 +28,21 @@ COPY build_info build_info/
 COPY middleware middleware/
 COPY template template/
 COPY utils utils/
+COPY mapper mapper/
+
+# Load AAGUID map
+RUN wget --quiet https://raw.githubusercontent.com/passkeydeveloper/passkey-authenticator-aaguids/main/aaguid.json -O aaguid.json
```

Comment:
It seems that way. I can bundle that file via `go-embed`. My reason for implementing it into the dockerfile was simple: I wanted to prevent us to create a commit simply to udpate the file.